libraries/AP_L1_Control/AP_L1_Control.h

```diff
@@ -13,6 +13,7 @@
  *  - Ability to use loiter radius smaller than L1 length
  */
 
+#include <AP_AHRS/AP_AHRS.h>
```

Comment:
We don't need this include - we don't need to know the shape of the class, only that it exists.

Instead, either pre-declare the class with `class AP_AHRS`, or modify the first use in the file (in the constructor) to be `class AP_AHRS &`